buttercup.el

```diff
@@ -580,10 +628,10 @@ KEYWORD can have one of the following values:
   (gethash (symbol-function spy)
            buttercup--spy-contexts))
 
-(defun buttercup--spy-calls-add (spy-function context)
+(defun buttercup--spy-calls-add (spy context)
```

Comment:
In other functions, `spy` refers to the symbol, not the function value. Here, it's the function value. Admittedly, the naming is not consistent, but I'd rather replace all occurrences of `spy` with the more explicit version than to conflate them even more.